projects/kubernetes/python/README.md

```diff
@@ -0,0 +1,198 @@
+# Running Python Application on Kubernetes
+
+## Kubernetes is an open source platform that offers deployment, maintenance, and scaling features. It simplifies management of containerized Python applications while providing portability, extensibility, and self-healing capabilities.
+
+## Whether Python application are simple or complex, Kubernetes lets you efficiently deploy and scale them seamlessly rolling out new features while limiting resources to only those required.
+
+### Pre-requistes
+
+-Docker
+-Kubectl
+-Sample Python App
+
+## Docker is an open platform to build and ship distributed applications. To install Docker, follow the Docker official documentation.
+
+## To verify that Docker runs your system:
```

Comment:
Docker is an open platform to build and ship distributed applications. To install Docker, follow the Docker official documentation.

To verify that Docker runs your system: